.github/labels.yml

```
@@ -1,38 +1,91 @@
-- name: bug
-  description: Something isn't working
+# The labels declared in this file are the ones avialables
+# https://github.com/ansys/ansys-sphinx-theme/labels
+
+# -- Defects and glitches labels----------------------------------------------
+
+- name: 'bug'
+  description: Defects or glitches reported by users or developers
   color: d42a34
 
-- name: dependencies
-  description: Related with project dependencies
-  color: ffc0cb
+- name: 'fix'
+  description:  Pull requests related to resolving problems or errors
+  color: d42a34
+
+# -- Style labels ------------------------------------------------------------
+
+- name: 'style'
+  description: Improvements related to general project style
+  color: dcbeff
+
+- name: 'style:code'
+  description: Improvements related to code style
+  color: dcbeff
+
+- name: 'style:docs'
+  description: Improvements related to doc style
+  color: dcbeff
+
+- name: 'style:branch-name'
+  description: Verifies branch name is compliant with PyAnsys guidelines
+  color: dcbeff
+
+- name: 'style:skip'
+  description: Skip style runs in CI/CD
+  color: dcbeff
+
+# -- Tests labels ------------------------------------------------------------
+
+- name: 'tests'
+  description: Related with improvements of the test suite
+  color: ffd8b1
+
+- name: 'tests:skip'
+  description: Skip tests runs in CI/CD
+  color: ffd8b1
+
+# -- Documentation labels ----------------------------------------------------
 
-- name: documentation
-  description: Improvements or additions to documentation
+- name: 'docs'
+  description: Issues related to documentation
   color: 0677ba
 
-- name: enhancement
-  description: New features or code improvements
-  color: FFD827
+- name: 'docs:api'
+  description: Related to API documentation
+  color: 0677ba
 
-- name: good first issue
-  description: Easy to solve for newcomers
-  color: 62ca50
+- name: 'docs:examples'
+  description: Related to documentation examples
+  color: 0677ba
 
-- name: maintenance
-  description: Package and maintenance related
-  color: f78c37
+- name: 'docs:skip'
+  description: Skip documentation runs in CI/CD
+  color: 0677ba
 
-- name: release
-  description: Anything related to an incoming release
-  color: ffffff
+# -- Dependencies labels -----------------------------------------------------
 
-- name: hacktoberfest
-  description: Hacktoberfest related
-  color: 62ca50
+- name: 'dependencies'
+  description: Related with project dependencies
+  color: fabed4
 
-- name: hacktoberfest-accepted
-  description: Hacktoberfest accepted contribution
-  color: 62ca50
+- name: 'dependencies:pyproject'
+  description: Related with project file dependencies
+  color: fabed4
+
+- name: 'dependencies:ci'
+  description: Related with pipelines dependencies
```

```suggestion
  description: Related to project file dependencies
  color: fabed4

- name: 'dependencies:ci'
  description: Related to pipelines dependencies
```
For consistency with other descriptions